docs/types/operators.rst

```diff
@@ -26,6 +26,23 @@ except for comparison operators where the result is always ``bool``.
 The operators ``**`` (exponentiation), ``<<``  and ``>>`` use the type of the
 left operand for the operation and the result.
 
+Ternary Operator
+----------------
+The ternary operator is used in expressions of the form ``<expression> ? <trueExpression> : <falseExpression>``.
+It evaluates one of the latter two given expressions depending upon the result of the evaluation of the main ``<expression>``.
+If ``<expression>`` evaluates to ``true``, then ``<trueExpression>`` will be evaluated, otherwise ``<falseExpression>`` is evaluated.
+
+The result of the ternary operator (``... ? ... : ...``) is never a number literal, even if all of its operands are number literals.
```

Comment:
```suggestion
The result of the ternary operator is never a number literal, even if all of its operands are number literals.
```
We don't need to remention the form of ternary expressions now that we started off with it.

But I'm also not sure if we should say that the result "is never a number literal"... That's a bit of a weird thing to say: a complex expression obviously isn't a literal... technically, it would be accurate to say "never has rational number type", but I'm not sure that really makes things clear, so I'm not sure what's best to do here...